chain/network/src/lib.rs

```diff
@@ -1,12 +1,12 @@
-#[macro_use]
-extern crate lazy_static;
-
-pub use peer::{EPOCH_SYNC_PEER_TIMEOUT_MS, EPOCH_SYNC_REQUEST_TIMEOUT_MS};
-pub use peer_manager::PeerManagerActor;
-pub use types::{
+pub use near_network_primitives::types::{
     FullPeerInfo, NetworkAdapter, NetworkClientMessages, NetworkClientResponses, NetworkConfig,
     NetworkRecipient, NetworkRequests, NetworkResponses, PeerInfo,
 };
+pub use peer::{EPOCH_SYNC_PEER_TIMEOUT_MS, EPOCH_SYNC_REQUEST_TIMEOUT_MS};
+pub use peer_manager::PeerManagerActor;
+
+#[cfg(feature = "metric_recorder")]
+pub use near_network_primitives::recorder;
```

Comment:
This seems to also need to be cleaned up

Reply:
Or maybe we want to keep the feature in the Cargo.toml file since there are other places where `metric_recorder` feature is used